lib/Logger.ts

```diff
@@ -1,9 +1,51 @@
-const fs = require('fs');
-const path = require('path');
-const winston = require('winston');
-const utils = require('./utils/utils');
+import fs from 'fs';
+import path from 'path';
+import winston from 'winston';
+import utils from './utils/utils';
+
+interface LoggerInterface {
+  level: number;
+  logDir: string;
+  context: any;
+}
+class Logger implements LoggerInterface {
+  static levels = {
+    ERROR: 0,
+    WARN: 1,
+    INFO: 2,
+    VERBOSE: 3,
+    DEBUG: 4,
+  };
+
+  static levelsVal = {
```

Comment:
Static properties were moved inside the class and the 'static' accessor was added.